src/main/java/racingcar/view/Console.java

```diff
@@ -24,19 +31,19 @@
         return carNames;
     }
 
-    public int setTryCount() {
+    public static int setTryCount() {
         System.out.println(INPUT_TRY_COUNT);
 
         return Integer.parseInt(scanner.nextLine());
     }
 
-    private void validateCarNameLength(String carName) {
-        if (carName.length() > 5) {
+    private static void validateCarNameLength(String carName) {
+        if (carName.length() > CAR_NAME_MAX) {
             throw new IllegalArgumentException(RacingCarException.CAR_NAME_INVALID);
```

Comment:
`RacingCarException`을 활용해보면 어떨까요?

```suggestion
            throw new RacingCarException(RacingCarException.CAR_NAME_INVALID);
```

Reply:
아이고.. 만들어두고 사용을 안했네요